src/main/java/edu/ksu/canvas/attendance/controller/CourseConfigurationController.java

```diff
@@ -120,7 +133,18 @@ public ModelAndView synchronizeWithCanvas(@PathVariable String sectionId) throws
     }
 
     @RequestMapping(value = "/{sectionId}/save", params = "pushGradesToCanvas", method = RequestMethod.POST)
-    public ModelAndView pushGradesToCanvas(@PathVariable Long sectionId, @ModelAttribute("courseConfigurationForm") @Valid CourseConfigurationForm classSetupForm, BindingResult bindingResult) throws NoLtiSessionException{
+    public ModelAndView pushGradesToCanvas(@PathVariable String sectionId, @ModelAttribute("courseConfigurationForm") @Valid CourseConfigurationForm classSetupForm, BindingResult bindingResult) throws NoLtiSessionException{
```

Comment:
maybe setting the sectionId as a Long would remove the need to convert it to Long in lne 164 and 165... if it's not being compared to a string or being used to set another object's string field in this method... I think that a canvas sectionId will always be a number, if it isn't then there is no way to recover from when a non number sectionId is submitted